test/functionalTest/cases/1426_empty_components_in_uri_path/double_slash_at_end_of_path.test

```diff
@@ -0,0 +1,566 @@
+Date: REGEX(.*)
+
+{
+    "description": "service not found",
+    "error": "BadRequest"
+}
+
+
+01b. /v2///
+===========
+HTTP/1.1 400 Bad Request
+Content-Length: 56
+Content-Type: application/json
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+{
+    "description": "service not found",
+    "error": "BadRequest"
+}
+
+
+02a. /v2/entities//
+===================
+HTTP/1.1 404 Not Found
+Content-Length: 95
+Content-Type: application/json
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+{
+    "description": "The requested entity has not been found. Check type and id",
+    "error": "NotFound"
+}
+
+
+02b. /v2/entities///
+====================
+HTTP/1.1 400 Bad Request
+Content-Length: 56
+Content-Type: application/json
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+{
+    "description": "service not found",
+    "error": "BadRequest"
+}
+
+
+03a. /v2/entities/E1//
+======================
+HTTP/1.1 400 Bad Request
+Content-Length: 56
+Content-Type: application/json
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+{
+    "description": "service not found",
+    "error": "BadRequest"
+}
+
+
+03b. /v2/entities/E1///
+=======================
+HTTP/1.1 400 Bad Request
+Content-Length: 56
+Content-Type: application/json
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+{
+    "description": "service not found",
+    "error": "BadRequest"
+}
+
+
+04a. /v2/entities/E1/attrs//
+============================
+HTTP/1.1 400 Bad Request
+Content-Length: 88
+Content-Type: application/json
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+{
+    "description": "attribute name length: 0, min length supported: 1",
+    "error": "BadRequest"
+}
+
+
+04b. /v2/entities/E1/attrs///
+=============================
+HTTP/1.1 400 Bad Request
+Content-Length: 56
+Content-Type: application/json
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+{
+    "description": "service not found",
+    "error": "BadRequest"
+}
+
+
+05a. /v2/entities/E1/attrs/A1//
+===============================
+HTTP/1.1 400 Bad Request
+Content-Length: 56
+Content-Type: application/json
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+{
+    "description": "service not found",
+    "error": "BadRequest"
+}
+
+
+05b. /v2/entities/E1/attrs/A1///
+================================
+HTTP/1.1 400 Bad Request
+Content-Length: 56
+Content-Type: application/json
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+{
+    "description": "service not found",
+    "error": "BadRequest"
+}
+
+
+06a. /v2/entities/E1/attrs/A1/value//
+=====================================
+HTTP/1.1 400 Bad Request
+Content-Length: 56
+Content-Type: application/json
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+{
+    "description": "service not found",
+    "error": "BadRequest"
+}
+
+
+06b. /v2/entities/E1/attrs/A1/value///
+======================================
```

Comment:
It seems to: https://github.com/telefonicaid/fiware-orion/pull/2314/files#diff-4c2a67b23b1d7b3b081ad601f6db3664R42

NTC